src/main/java/com/microfocus/octane/plugins/views/CoverageUiHelper.java

```diff
@@ -127,6 +131,14 @@ private static MapBasedObject convertGroupEntityToUiEntity(TestStatusDescriptor
         return groups;
     }
 
+    private static void addSkippedToRequiresAttention(Map<String, GroupEntity> statusId2group) {
```

Comment:
unclean code, adds but also remove the group.
removal should be separated, or method name should reflect what it does (don't recommend)